### ISSUE TITLE
[REQUEST] maximum and minimum value with decimal places

### ISSUE THREAD

**Issue by DanSilva926** [agronick/aa-torque#72]
I come to suggest that in the field where we define the maximum and minimum value, there is the possibility of defining decimal values, for example, in my car the maximum turbo boost is 1.55 bar, I cannot define the maximum as 1.6, it just accepts or 1 bar or 2 bars

**Comment by DanSilva926** [comment]
![aatorque](https://github.com/agronick/aa-torque/assets/92631888/a05ecc75-ca4d-41ea-a19d-8ba6048fd65e)

where it says max thanks, it says 2, but I can't put 1.6, because it only accepts 1 or 2, it doesn't accept decimal places

**Comment by agronick** [comment]
Yeah that was kind of an oversight. Someone is welcome to make a pr to fix it. It's not super critical to me. There are a bunch of places that data touches where it needs to be switched from integer to float. You can convert it to millibar if you want.
